pkgs/top-level/all-packages.nix

```diff
@@ -28320,6 +28320,7 @@ in
 
   zola = callPackage ../applications/misc/zola {
     inherit (darwin.apple_sdk.frameworks) CoreServices;
+    rustPlatform = rustPackages_1_45.rustPlatform;
```

Comment:
How about adding a comment. Something like
`version 0.13.0 has The lexical_core 0.7.4 dependency that doesn't build with rust 1.53. This should be fixed in the next version`

Reply:
good idea